.github/workflows/static_checks.yml

```diff
@@ -20,3 +20,56 @@ jobs:
         with:
           checks: "duppatterns,files,syntax"
           experimental_checks: "notowned"
+
```

Comment:
Can we add this above

```
# This workflow has the following jobs:
# Check Codeowners
# linters: Java, Kotlin, Protobuf, Bazel
```

Reply:
Can you clarify the purpose of doing this? I know we did it earlier, but it seems fairly easy to determine from a glance or from GitHub itself which jobs are run, and this list easily becomes out-of-date.

Reply:
Yes, this seems like require updation all the time, let's skip this.